docs/reference/requirement-sets/excel-api-online-requirement-set.md

```diff
@@ -35,7 +35,23 @@ Once the API is in a cross-platform requirement set, you should remove or edit t
 
 ## API list
 
-There are currently no online-only APIs. Check back as new features are added to Excel on the web and supported by the Office JavaScript APIs.
+The following APIs are currently available for Excel on the web as part of the `ExcelApiOnline 1.1` requirement set.
+
+| Class | Fields | Description |
+|:---|:---|:---|
+|[Comment](/javascript/api/excel/excel.comment)|[mentions](/javascript/api/excel/excel.comment#mentions)|Gets the entities (e.g. people) that are mentioned in comments.|
+||[richContent](/javascript/api/excel/excel.comment#richcontent)|Gets the rich comment content (e.g. mentions in comments). This string is not meant to be displayed to end-users. Your add-in should only use this to parse rich comment content.|
+||[updateMentions(contentWithMentions: Excel.CommentRichContent)](/javascript/api/excel/excel.comment#updatementions-contentwithmentions-)|Updates the comment content with a specially formatted string and a list of mentions.|
+|[CommentMention](/javascript/api/excel/excel.commentmention)|[email](/javascript/api/excel/excel.commentmention#email)|Gets or sets the email address of the entity that is mentioned in comment.|
+||[id](/javascript/api/excel/excel.commentmention#id)|Gets or sets the id of the entity. This is aligned with the id information in `CommentRichContent.richContent`.|
```

Comment:
What does "is aligned with" mean in this context? Does this mean "exactly matches" or something else?